homeassistant/components/coronavirus/config_flow.py

```diff
@@ -15,13 +19,18 @@ class ConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
 
     _options = None
 
-    async def async_step_user(self, user_input=None):
+    async def async_step_user(
+        self, user_input: dict[str, Any] | None = None
+    ) -> dict[str, Any]:
```

Comment:
```suggestion
    ) -> FlowResultDict:
```

Reply:
I didn't use that on purpose, so it could be part of a patch. If I would apply this change, it cannot be shipped as a patch.

Reply:
Right.